query.go

```diff
@@ -39,7 +39,7 @@ type Selector struct {
 // node(s) matching the criteria.
 //
 // Query actions that target a browser DOM element node (or nodes) make use of
-// Query, in conjunction with the After option (see below) to retrieve data or
+// Query, in conjunction with the After option to retrieve data or
```

Comment:
This block of comment can be modified to add more doc links. For example:

- `After` -> `[After]`
- `Tasks` -> `[Tasks]`
- `AtLeast` -> `[AtLeast]`
- `BySearch` -> `[BySearch]`
- `ByID` -> `[ByID]`
- `ByQuery` -> `[ByQuery]`
- `ByQueryAll` -> `[ByQueryAll]`
- ...